### ISSUE TITLE
Data Breach Report fails due to CORS Header

### ISSUE THREAD

**Issue by iamtpage** [bitwarden/web#262]
When I click on `Check Breaches` it fails due to CORS:

```
Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at 
https://haveibeenpwned.com/api/v2/breachedaccount/<email address removed>. (Reason: missing token 
‘cache-control’ in CORS header ‘Access-Control-Allow-Headers’ from CORS preflight channel).
```

FF 62 is my browser version

**Comment by kspearrin** [comment]
This was fixed in 2.3.0. This on a older self-hosted version?

**Comment by iamtpage** [comment]
Has 2.3.0 been released? Under the releases tab I see 2.2.0 is the latest

**Comment by kspearrin** [comment]
It's been released on vault.bitwarden.com . Will be released elsewhere in time.
